exo-go/features/test/test-service.feature

```diff
@@ -0,0 +1,23 @@
+Feature: testing an Exosphere service
+
+  As an Exosphere developer working with a variety of services written in different languages
+  I want to have a standardized way to run all tests for a service
+  So that I don't have to waste time investigating how to use language-specific tools.
+
+  Rules:
+  - when running "exo test" in a service directory, it runs the test for the respective service
+
+  Scenario: testing a service
+    Given I am in the "exoservice" directory of the "tests-cucumber" example application
```

Comment:
`test-cucumber` isn't a good example for this since it contains a single service. I'd suggest using something like the tests-passing application, where you run in one of the services. We should also validate it does not run tests in any other service.